packages/block-editor/src/components/block-popover/index.js

```diff
@@ -88,6 +93,20 @@ function BlockPopover(
 
 		return {
 			getBoundingClientRect() {
+				if ( isZoomOut() ) {
+					const selectedRect =
```

Comment:
```suggestion
					const selectedBlockRect =
```

Nit: would be easier to read.